llvm/include/llvm/BinaryFormat/ELFRelocs/RISCV.def

```diff
@@ -55,3 +55,7 @@ ELF_RELOC(R_RISCV_SET32,             56)
 ELF_RELOC(R_RISCV_32_PCREL,          57)
 ELF_RELOC(R_RISCV_IRELATIVE,         58)
 ELF_RELOC(R_RISCV_PLT32,             59)
+ELF_RELOC(R_RISCV_TLSDESC_HI20,      62)
```

Comment:
I think this needs to be rebased. There should be a 60 and 61 defined.

Reply:
Yes, it does. Did we resolve how to do that w/ PRs? I recall there was some concerns about the rebase causing issues with code review.

My understanding is that `git rebase main` and pushing to my branch will erase much of the comments and review history. Do you know if there is a good way to rebase w/o losing the review comments? I've been following https://discourse.llvm.org/t/force-push-and-rebase/73766 but I don't see a concrete reccomendation in the thread.